src/server/log.ml

```diff
@@ -68,11 +68,6 @@ let logs_lib_tag : string Logs.Tag.def =
     request_id_label
     Format.pp_print_string
 
-(* Lwt sequence-associated storage key used to pass request ids for use when
-   ~request is not provided. *)
-let id_lwt_key : string Lwt.key =
-  Lwt.new_key ()
```

Comment:
Eio now has [Eio.Fiber.create_key](https://ocaml-multicore.github.io/eio/eio/Eio/Fiber/index.html#val-create_key) for this